accesstoken/src/test/java/uk/gov/di/ipv/cri/common/api/handler/pact/utils/MockHttpServer.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.di.ipv.cri.common.api.handler.pact.utils;
+
+import com.sun.net.httpserver.HttpServer;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.util.ArrayList;
+import java.util.concurrent.Executors;
+
+public class MockHttpServer {
+
+    public static int PORT = 5050;
+
+    public static void startServer(ArrayList<Injector> endpointList) throws IOException {
```

Comment:
As this is used by the test and the port is defined there, we can actually remove `public static int PORT = 5050;` and update this method to accept a port parameter. `startServer(ArrayList<Injector> endpointList, int port)`

Reply:
Which does mean 
```
String pactPort = System.getenv("PACT_PORT");
        if (pactPort != null) {
            PORT = Integer.parseInt(pactPort);
        }
```
 can be removed